swarm.cabal

```diff
@@ -158,6 +160,7 @@ library
                       prettyprinter                 >= 1.7.0 && < 1.8,
                       random                        >= 1.2.0 && < 1.3,
                       servant                       >= 0.19 && < 0.20,
+                      servant-docs                  >= 0.12 && < 0.13,
```

Comment:
Not sure why the GHC 9.4 CI is failing, since `servant-docs` [added support for GHC 9.4 in October](https://github.com/haskell-servant/servant/pull/1592).

Reply:
@TristanCacqueray I see you were involved with bumping the upper bound for lens [here](https://github.com/haskell-servant/servant/pull/1614).  However, the lens bound for [`servant-docs` on Hackage is still `< 5.2`](https://hackage.haskell.org/package/servant-docs).  Do you know how that can be fixed?

Reply:
I think the maintainers need to do a new release or revision, e.g. https://hackage.haskell.org/package/servant-docs-0.12/revisions/ does not include the recent bump.

Reply:
https://github.com/haskell-servant/servant/issues/1636

Reply:
Looks like the maintainers just made a new `servant-docs` release.

Reply:
Yes, the build now succeeds.